src/controllers/news.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
+require('dotenv').config();
+
+router.get('/sources', (req, res, next) => {
+    console.log(`https://newsapi.org/v2/top-headlines/sources?apiKey=${process.env.API_KEY}`);
+    fetch(`https://newsapi.org/v2/top-headlines/sources?apiKey=${process.env.API_KEY}`, {method: 'get'})
```

Comment:
do you need to specify the get method here `{method: 'get'}` wouldn't just `fetch(url).then` work?